src/Tests/CaptchaAdminTestCase.php

```diff
@@ -252,6 +252,10 @@ public function testCaptchaPlacementCacheClearing() {
     // Check that the placement cache is unset.
     $placement_map = $this->container->get('cache.default')->get('captcha_placement_map_cache');
     $this->assertFalse($placement_map, 'CAPTCHA placement cache should be unset after cache clear.');
+    // Check for cache results.
+    $this->drupalLogout();
+    $this->drupalGet('user/login');
+    $this->assertFalse($this->drupalGetHeader('x-drupal-cache') == 'HIT', 'Captcha cache has been deleted.');
```

Comment:
This isn't related to cache clearing, so place it in a different test. Also, assertFalse() should be easier. You might also want to add a check somewhere where captcha is disabled to see a HIT/MISS.